schema.yml

```diff
@@ -0,0 +1,224 @@
+    patch:
+      description: 'Mark a single given notification as read if the notification is
+        sent. If the notification is already mark as sent, it marks it as sent.'
+      operationId: v1_partial_update
+      parameters:
+      - in: path
+        name: notification_uuid
+        required: true
+        schema:
+          description: ''
+          title: ''
+          type: string
+      - in: query
+        name: ordering
+        schema:
+          description: Which field to use when ordering the results.
+          title: Ordre
+          type: string
+      - in: query
+        name: search
+        schema:
+          description: A search term.
+          title: Rechercher
+          type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Notification'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+      tags:
+      - v1
+    put:
+      description: 'Mark a single given notification as read if the notification is
+        sent. If the notification is already mark as sent, it marks it as sent.'
+      operationId: v1_update
+      parameters:
+      - in: path
+        name: notification_uuid
+        required: true
+        schema:
+          description: ''
+          title: ''
+          type: string
+      - in: query
+        name: ordering
+        schema:
+          description: Which field to use when ordering the results.
+          title: Ordre
+          type: string
+      - in: query
+        name: search
+        schema:
+          description: A search term.
+          title: Rechercher
+          type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Notification'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+      tags:
+      - v1
+  /notifications/v1/mark_all_as_read:
+    put:
+      description: 'Mark all the current user sent notifications as read.'
+      operationId: v1_mak_all_as_read
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotificationList'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+      tags:
+      - v1
+servers:
+  - url: https://{environment}.osis.uclouvain.be/
+    variables:
+      environment:
+        default: dev
+        enum:
+          - dev # Development server
+          - qa # Staging server
+          - test # Test server
+  - url: https://osis.uclouvain.be/
```

Comment:
Tu peux remplacer en '**https://osis.uclouvain.be/api/v1/notification/**' et donc supprimer pour tous les autres endpoints (prefix: /notifications/v1)